gspread/models.py

```diff
@@ -778,15 +783,21 @@ def get_all_records(
             return []
 
         keys = data[idx]
-        values = [
-            numericise_all(
-                row,
-                empty2zero,
-                default_blank,
-                allow_underscores_in_numeric_literals,
-            )
-            for row in data[idx + 1:]
-        ]
+
+        if numericise_ignore == ['all']:
+            values = [row for row in data[idx + 1]]
```

Comment:
It looks like in this case it only iterates over one row. Shouldn't it be `data[idx + 1:]` like in an `else` clause?